recipes-ros/ros/files/0001-use-env-to-get-python-exec.patch

```diff
@@ -0,0 +1,32 @@
+From 4254977bbfc67a3083e3881a56b50b729bf57d9f Mon Sep 17 00:00:00 2001
+From: Dominique Hunziker <dominique.hunziker@gmail.com>
+Date: Fri, 2 Jun 2017 11:57:54 +0200
+
+Upstream-Status: Pending
+
+---
+ core/roslib/env-hooks/10.ros.sh.em | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/core/roslib/env-hooks/10.ros.sh.em b/core/roslib/env-hooks/10.ros.sh.em
+index caed342..1c64fc4 100644
+--- a/core/roslib/env-hooks/10.ros.sh.em
++++ b/core/roslib/env-hooks/10.ros.sh.em
+@@ -1,7 +1,7 @@
+ # generated from ros/env-hooks/10.ros.sh.em
+ 
+ # scrub old ROS bin dirs, to avoid accidentally finding the wrong executables
+-export PATH="`@(PYTHON_EXECUTABLE) -c \"import os; print(os.pathsep.join([x for x in \\\"$PATH\\\".split(os.pathsep) if not any([d for d in ['cturtle', 'diamondback', 'electric', 'fuerte'] if d in x])]))\"`"
++export PATH="`/usr/bin/env python -c \"import os; print(os.pathsep.join([x for x in \\\"$PATH\\\".split(os.pathsep) if not any([d for d in ['cturtle', 'diamondback', 'electric', 'fuerte'] if d in x])]))\"`"
```

Comment:
This should be fine, but maybe better to be explicit about which version to use.